### DISCUSSION TITLE
Minimum Android SDK poll

### DISCUSSION THREAD

**Discussion by fondesa** [androiddevelopersitalia/aditogether#30] [Polls]
I listed here only SDKs with some relevant changes.

**Comment by giuseppesorce** [comment]
We can use 26 as min SDK without problems

**Reply by fondesa** [reply]
Personally I think 26 is too high. Having 26 as the min SDKs, it means we are excluding 10% of the users.
Also, since this project should also show best practices, filtering out older versions is a missed opportunity to show a compat behavior across different app versions IMHO.

Anyway I can't update an already existing poll unfortunately since it would be reset. If 23 wins, I can open another poll with 23 as the minimum.

**Reply by giuseppesorce** [reply]
I think that the correct  minSdk depends from target of app. Every product has his target and his users target so we can use 26 as minSdk without losing any users because our users use  recentely devices.

**Reply by fondesa** [reply]
I completely agree on:
> the correct minSdk depends from target of app

What I meant though is that this app IMHO should be seen also as a technical exercise on best practices not only as the final product.
The same applies also on choices like resources for tablets (or a wear module and multiple other things).
The users of this app surely won't care about using it from a tablet or a watch but having them included would show the best practices when these options are present in a real scenario.

To be clear, decreasing the min sdk version, increases the difficulty, but this is (should at least for me) be the aim of this project: showing how to handle complex situations which can be present in a real world scenario

**Reply by fardavide** [reply]
I always take this as a Bible 😁

https://androidmanife.st/@minSdkVersion/109781794410800323

I don't fully share the point about complexity, to me it's more a matter of using outdated tools 🙂 I compare it to using Eclipse or Java 😛 (14 is Assembly level tho 😁)

**Comment by fondesa** [comment]
The clear winner is 23. Let's close this and in case re-open another one with higher SDK choices!
Temporarily, the project is using 23 as min sdks
